src/components/RockPaperScissors/styles/Scoreboard.styled.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+
+export const StyledIconTitle = styled.img<{ imgSize?: string }>`
+  height: ${(props) => props.imgSize ?? '2rem'};
+  width: ${(props) => props.imgSize ?? '2rem'};
+`
+export const StyledScoreboard = styled.div`
+  width: 100%;
+  max-width: 800px;
+  //min-height: 150px;
+  border-radius: 5px;
+  border: 2px solid #4e5f7d;
```

Comment:
border can be increased slightly to maybe 3 and border-radius can be increase to maybe around 10px to look more like the original?